osf/models/node.py

```diff
@@ -2425,6 +2425,10 @@ class Node(AbstractNode):
     FYI: Behaviors common between Registration and Node should be on the parent class.
     """
 
+    @property
+    def MachineClass(self):
+        raise NotImplementedError()
+
```

Comment:
This is a fail-safe to prevent nodes from being entered into the moderation process. Because Nodes/Projects and Registration are actually all the same table, but separate models, this ensures nobody will run any of the registration machine action on it if they only are a aware they have an AbstractNode, but know that I think about it more... 

... since the actual machinemixin isn't on the AbstractNode anymore, but instead on the DraftRegistration, this isn't at all necessary. Good catch!